book/04-git-server/sections/protocols.asc

```diff
@@ -56,10 +56,12 @@ If you and a co-worker are working on the same project and they want you to chec
 The cons of this method are that shared access is generally more difficult to set up and reach from multiple locations than basic network access.
 If you want to push from your laptop when you're at home, you have to mount the remote disk, which can be difficult and slow compared to network-based access.
 
-It's also important to mention that this isn't necessarily the fastest option if you're using a shared mount of some kind.
+It's important to mention that this isn't necessarily the fastest option if you're using a shared mount of some kind.
 A local repository is fast only if you have fast access to the data.
 A repository on NFS is often slower than the repository over SSH on the same server, allowing Git to run off local disks on each system.
 
+Finally, this protocol does not protect the repository against mistaken damages, since it is accessible as any standard directory to anyone having access to it.
```

Comment:
sounds perfect!

I will update the PR and notify when done.

On Tue, May 26, 2015 at 9:22 AM, Ben Straub <notifications@github.com>
wrote:

> In book/04-git-server/sections/protocols.asc
> <https://github.com/progit/progit2/pull/308#discussion_r31009317>:
>
> >  A local repository is fast only if you have fast access to the data.
> >  A repository on NFS is often slower than the repository over SSH on the same server, allowing Git to run off local disks on each system.
> >
> > +Finally, this protocol does not protect the repository against mistaken damages, since it is accessible as any standard directory to anyone having access to it.
>
> Yeah, we're getting a lot closer now. Just one rewording suggestion:
>
> Finally, this protocol does not protect the repository against accidental
> damage. Every user has full shell access to the "remote" directory, and
> there's nothing preventing them from changing or removing internal Git
> files and corrupting the repository.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/progit/progit2/pull/308/files#r31009317>.
>